TangoRosStreamer/app/src/main/res/layout/dialog_load_nav_map.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2017 Intermodalics All Rights Reserved. -->
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Consider using an `AlertDialog` and autocreate a list with existing map names on the system.

An AlertDialog can show a title, up to three buttons and a list of selectable items (or a custom layout if you would still like to do something special).